bin/ch/ch.cpp

```diff
@@ -769,6 +769,11 @@ int _cdecl wmain(int argc, __in_ecount(argc) LPWSTR argv[])
         ChakraRTInterface::UnloadChakraDll(chakraLibrary);
     }
 
+    if (ttUri != nullptr)
+    {
+        free(ttUri);
```

Comment:
Sorry, not sure I follow- the allocation and freeing of memory is done by the ch module using a function in codex which is always linked as static lib with ch so it should be fine. I think Curtis' point is regarding ChakraCore dynamic library (that is, we should treat the ChakraCore static lib case the same as if it was a dynamic library). Codex is always static so should be fine.